Virtual Librarian/Virtual Librarian/Library.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Virtual_Librarian
+{
+    public sealed class Library
+    {
+        private static Library instance = null;
+        private static readonly object padLock = new object();
+        Library() { }
+        public static Library Instance
+        {
+            get
+            {
+                lock (padLock)
+                {
+                    if (instance == null)
+                    {
+                        instance = new Library();
+                    }
+                    return instance;
+                }
+            }
+        }
+
+        private List<Book> allBooks = new List<Book>();
+        public void addBook(Book book)
```

Comment:
Please use consistent method naming -> SomeMethod (PascalCase, standart for methods in C#)